src/printf.py

```diff
@@ -5,6 +5,7 @@
 # Using the printf operator (%), print the following feeding in the values of x,
 # y, and z:
 # x is 10, y is 2.25, z is "I like turtles!"
+print("x is %d, y is %.2f, z is %s" % (x, y, z))
 
-
-# Use the 'format' string method to print the same thing
\ No newline at end of file
+# Use the 'format' string method to print the same thing
+print(f"x is {x}, y is {round(y, 2)}, z is {z}")
```

Comment:
Were actually looking for the .format method, would take a crack at it that way?